tests/test_parser.py

```diff
@@ -7,17 +8,29 @@
 from streaming_form_data.targets import ValueTarget
 
 
-DATA_DIR = 'tests/data'
+def get_random_bytes(size, seed):
+    random.seed(seed)
+    return random.bytes(size)
 
 
-def data_file_path(filename):
-    return os.path.join(DATA_DIR, filename)
+def data_file_open(filename):
```

Comment:
This somehow makes me a little bit confused. This function is not opening any files, but returning random data, but also giving the caller of this function that a file is being opened.

Proposal: either let's rename this function according to what it does (and does not), or keep the existing tests the way they are (i.e., calls to `data_file_get_multipart('example.txt')` return contents of `example.txt` instead of random data).

Thoughts?